### DISCUSSION TITLE
Hybrid app (react-native and ember webView)

### DISCUSSION THREAD

**Discussion by thaole2021** [wix/Detox#2946] [Q&A]
How to interact detox with a WebView element? My company app is using react-native and ember ascend WebView. I couldn’t get detox working with element on WebView for IOS testing. 
Any help would be appreciated! 
If detox doesn’t support WebView, does anyone know a work-around so detox can interact with WebView?
Thanks.

**Comment by phuongwd** [comment]
@thaole2021 It support only on Android and not avaiable on documents.
You can refer there https://github.com/wix/Detox/blob/master/detox/test/e2e/29.webview.test.js

**Reply by thaole2021** [reply]
Thanks Phuong. I will use this for my test.

**Reply by GhayoorUlHaq** [reply]
its not working 🙁. Facing error **ReferenceError: web is not defined**

**Reply by thaole2021** [reply]
It is working for me. What is the detail error that you got and what is your element?
